app/src/main/java/com/zeapo/pwdstore/crypto/PasswordCreationActivity.kt

```diff
@@ -279,166 +279,179 @@ class PasswordCreationActivity : BasePgpActivity(), OpenPgpServiceConnection.OnB
     /**
      * Encrypts the password and the extra content
      */
-    private fun encrypt(receivedIntent: Intent? = null) = with(binding) {
-        val editName = filename.text.toString().trim()
-        val editPass = password.text.toString()
-        val editExtra = extraContent.text.toString()
-
-        if (editName.isEmpty()) {
-            snackbar(message = resources.getString(R.string.file_toast_text))
-            return@with
-        } else if (editName.contains('/')) {
-            snackbar(message = resources.getString(R.string.invalid_filename_text))
-            return@with
-        }
+    private fun encrypt(receivedIntent: Intent? = null) {
+        with(binding) {
```

Comment:
I had to move this here because type checking was failing when I tried to call encrypt recursively.